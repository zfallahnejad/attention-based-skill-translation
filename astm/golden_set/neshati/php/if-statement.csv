250259,if-statement
476,if-statement
541091,if-statement
1491895,if-statement
187606,if-statement
218196,if-statement
1315962,if-statement
50079,if-statement
862594,if-statement
