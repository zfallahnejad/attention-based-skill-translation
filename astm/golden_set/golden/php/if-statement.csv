250259,20,23,0.46511627906976744
476,20,29,0.40816326530612246
1491895,13,18,0.41935483870967744
541091,13,9,0.5909090909090909
187606,10,6,0.625
218196,10,7,0.5882352941176471
1315962,8,5,0.6153846153846154
50079,7,10,0.4117647058823529
862594,7,3,0.7
